lib/Echidna/Transaction.hs

```diff
@@ -116,7 +116,7 @@ canShrinkTx Tx { call, gasprice = 0, value = 0, delay = (0, 0) } =
 canShrinkTx _ = True
 
 removeCallTx :: Tx -> Tx
-removeCallTx t = Tx NoCall 0 t.src 0 0 0 t.delay
+removeCallTx t = Tx NoCall t.src t.dst 0 0 0 t.delay
```

Comment:
Why this was changed?

Reply:
`t.src` was in the wrong place and having 0 in the src caused a crash in tx generation which was previously not triggered as there was something deployed on the address 0. I think this was changed in hevm and now there is nothing deployed on 0.